examples/sys/complex/3rdparty/crates/BUILD.vcpkg-0.2.15.bazel

```diff
@@ -35,9 +33,44 @@ rust_library(
     rustc_flags = ["--cap-lints=allow"],
     tags = [
         "cargo-bazel",
+        "crate-name=vcpkg",
         "manual",
         "noclippy",
         "norustfmt",
     ],
+    target_compatible_with = select({
```

Comment:
This block gets repeated in a lot of places. Is putting it somewhere common and just referencing it from each of these a viable option to reduce the number of places it need to be (and the number of changes needed when the list changes)?

Reply:
No - BUILD files are supposed to not follow the DRY principle.  See https://rsvp.withgoogle.com/events/bazelcon2023/sessions/day2-talk5

If you were to use a variable instead, then it becomes a lot harder for tooling to do analysis.

Reply:
SG, thanks.